handbook/process/copyright.md

```diff
@@ -0,0 +1,37 @@
+The FPA can, if required by a contributor, assume the copyright over a contribution made by that contributor.
+
+The reason might be:
+
+1. **Give the FPA full control over the destiny of that file**
+
+The FPA and its administrators will then be able to do things like change the file's license without the author's consentment. It should be noted that this action concentrates copyright in the hands of a single body, which is not something advisable in a community-developed project. The FreeCAD code being in many hands is actually beneficial for the software, as it prevents a single actor from gaining too much power over the code. As protecting the community nature of FreeCAD is part of the statutes of the FPA, contributors are encouraged to not hand over their contributions to the FPA without a good reason. Their individual hold over part of the FreeCAD code's copyright is a strength to FreeCAD.
```

Comment:
I don't think the discussion of copyright concentration belongs here.  It feels like it needs the wikipedia *citation needed.  There are a lot of schools of thought on that topic and I don't think we want to pick one.

I would rather there be a simple explanation of what it means to assign copyright.  What is the creator giving up.
Related to that, It's appropriate to touch on why someone might _want_ to assign copyright so I would leave that part alone.

Reply:
Not sure... I think this should inform users of what they are doing. They will likely not go see elsewhere. However you have a point, this is a single point of view. What about adding more, like reasons why you WOULD want to hand you r copyright over